packages/ui/src/components/select/NetworkSelection.tsx

```diff
@@ -34,22 +34,26 @@ const NetworkSelection = () => {
   )
 
   const renderNetworks = useCallback(
-    (allowedNetworks: string[]) =>
-      networksToShow
-        .filter(([networkName]) => allowedNetworks.includes(networkName))
-        .map(([networkName, { logo }]) => (
-          <MenuItemStyled
-            key={networkName}
-            value={networkName}
-            data-cy={`select-network-option-${networkName}`}
-          >
-            <ImgStyled
-              alt={`network-logo-${networkName}`}
-              src={logo}
-            />
-            <ItemNameStyled>{networkName}</ItemNameStyled>
-          </MenuItemStyled>
-        )),
+    (allowedNetworks: string[]) => {
+      const filteredNetworks = networksToShow.filter(([networkName]) =>
+        allowedNetworks.includes(networkName)
+      )
```

Comment:
My opinion, but I don't think it helps in understanding anything. 
Having `networksToShow.fiter.map` is the same but shorter than `const fileteredNetwowk... and then filteredNetwork.map ` :shrug: 

Maybe that's a naming issue with "filteredNetworks", "networksToShow", "allowedNetworks" not being super meaningful. In any case this change adds more lines to no help in the comprehension IMHO.

Reply:
I split it as I need to know the amount of networks to apply CSS styles:
```js
isOneLine={filteredNetworks.length === 1}
```

Reply:
ah my bad. Then let's rename to `displayedNetworks` maybe?